java/vector/src/main/java/org/apache/arrow/vector/ipc/ArrowFileReader.java

```diff
@@ -140,21 +137,6 @@ public Map<String, String> getMetaData() {
     return new HashMap<>();
   }
 
-  /**
-   * Read a dictionary batch from the source, will be invoked after the schema has been read and
-   * called N times, where N is the number of dictionaries indicated by the schema Fields.
-   *
-   * @return the read ArrowDictionaryBatch
-   * @throws IOException on error
-   */
-  public ArrowDictionaryBatch readDictionary() throws IOException {
```

Comment:
We're going to need to mark this as a breaking change.

Reply:
Ah, good catch, thanks.